panel/io/fastapi.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
```

Comment:
Have you considered whether there would be a more FastAPI like way of integrating Panel?

Something that would be very understandable to users of FastAPI? Where Panel is more seen as one component of many in a larger FASTAPI server?

Reply:
Something that would make sense in the head of Sebastian Ramirez and align with how other components integrate?

Reply:
Communication wise its much more valuable to say Panel now provides a FastAPI plugin or component than trying to explain Panel runs on FastAPI but in its own unique way.

Reply:
I am honestly not quite following what your suggestion is. You can add Panel apps to an existing FastAPI app, so not sure what else it is you're asking for.

Reply:
Ok. I guess to need the documentation for how this is used.

Reply:
Hope the docs here clarified things.